apps/web/src/blocks/Layout/index.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+
+import React, { PropsWithChildren } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { css, styled } from 'styled-components';
+
+import Button from 'ui/components/Button';
+
+import { Layout as LayoutType, Media } from '@web/payload/payload-types';
+import isExpandedDoc from '../../lib/isExpandedDoc';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px;
+  ${({
+    theme: {
+      mq: { lg }
+    }
+  }) => lg`
+    flex-direction: row;
+    gap: unset;
+  `}
+`;
+
+const MenuLinksContainer = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 28px;
+`;
+
+export const MenuLink = styled.p`
```

Comment:
I think you'll want to create a styled component that wraps the `Link` rather than this